src/scippnexus/nxdata.py

```diff
@@ -603,13 +603,13 @@ def assemble(self, dg: sc.DataGroup) -> Union[sc.DataArray, sc.Dataset]:
     @staticmethod
     def transform_bitmask_to_dict_of_masks(bitmask: sc.Variable, suffix: str = ''):
         bit_to_mask_name = {
-            0: 'gap',
-            1: 'dead',
-            2: 'under_responding',
-            3: 'over_responding',
-            4: 'noisy',
+            0: 'gap_pixel',
+            1: 'dead_pixel',
+            2: 'under_responding_pixel',
+            3: 'over_responding_pixel',
```

Comment:
```suggestion
            2: 'underresponding_pixel',
            3: 'overresponding_pixel',
```
I think these are actually single words?

Reply:
In the nexus standard they are separated by space. Do you think we should join them anyway?

Reply:
I'll join them. It does look better.